web/modules/custom/rsvplist/src/Controller/ReportController.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Drupal\rsvplist\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * ReportController class to process the routes.
+ */
+class ReportController extends ControllerBase {
+
+  /**
+   * Function to load the RSVP submission datas
+   * that includes the user name of the user submitting the form
+   * Node id of the node
+   * and the email id the form was submitted with.
+   *
+   * @return array|null
+   *   Returns the array.
```

Comment:
Don't just mention the datatype provide a proper description.

Reply:
@Vishalprasad28 ?